dotcom-rendering/src/client/userFeatures/cookies/cookieHelpers.ts

```diff
@@ -0,0 +1,30 @@
+import { removeCookie, setCookie } from '@guardian/libs';
+import { AD_FREE_USER_COOKIE } from './adFree';
+import { ALLOW_REJECT_ALL_COOKIE } from './allowRejectAll';
+import { HIDE_SUPPORT_MESSAGING_COOKIE } from './hideSupportMessaging';
+import { USER_BENEFITS_EXPIRY_COOKIE } from './userBenefitsExpiry';
+
+export const timeInDaysFromNow = (daysFromNow: number): number => {
+	const tmpDate = new Date();
+	tmpDate.setDate(tmpDate.getDate() + daysFromNow);
+	return tmpDate.getTime();
+};
+
+export const extendCookieExpiry = (
+	cookieName: string,
+	daysTillExpiry: number = 1,
+): void => {
+	const expiresValue = timeInDaysFromNow(daysTillExpiry);
+	setCookie({
+		name: cookieName,
+		value: expiresValue.toString(),
```

Comment:
Am I right in thinking the value is never used by DCR?

Reply:
Yes that's right, we have to give it some value so I thought I'd put the expiry value in there even though we don't really use it. If you have a better suggestion though I'm open to changing it.